Classes/Package/CustomPages.php

```diff
@@ -4,26 +4,25 @@
 
 class CustomPages {
 
-
 	public $prefix        = 'cp';
 	public $special_pages = '';
 
 	public function __construct() {
 		$this->special_pages = [
-			'selfservice' => __( 'Self-Service', 'sha' ),
-			'vzughome'    => __( 'V-ZUG-Home', 'sha' ),
-			'contact'     => __( 'Kontakt', 'sha' ),
-			'search'      => __( 'Search', 'sha' ),
-			'404'         => __( '404 Error', 'sha' ),
+			'contact'   => __( 'Kontakt', 'sha' ),
+			'search'    => __( 'Search', 'sha' ),
+			'404_error' => __( '404 Error', 'sha' ),
```

Comment:
Begriffe bitte sprachlich einheitlichen.
```suggestion
			'404_error' => __( '404-Fehler', 'sha' ),
```